inbm/dispatcher-agent/dispatcher/sota/constants.py

```diff
@@ -14,6 +14,13 @@
 # Mender artifact path
 MENDER_ARTIFACT_PATH = get_canonical_representation_of_path("/etc/mender/artifact_info")
 
+# Update Tool file path
+UPDATE_TOOL_PATH = get_canonical_representation_of_path('/usr/bin/os-update-tool.sh')
```

Comment:
on re-review, I think this would be clearer as `TIBER_UPDATE_TOOL_PATH`

Reply:
Constant name updated.